plugins/modules/accesspoint_workflow_manager.py

```diff
@@ -3333,6 +3935,94 @@ def reboot_factory_reset_function(self, ap_list, reboot_or_reset):
 
             return self
 
+    def bulk_ap_update(self, bulk_config):
+        """
+        Access Point bulk update common field to update to all mentioned AP list.
+
+        Parameters:
+            self (dict): A dictionary used to collect the execution results.
+            bulk_config (dict): A dict containing the APs mac address and cofig details which need to update APs.
+
+        Returns:
+            dict: A dictionary containing the result of the buld access point update status.
+        """
+        ap_exist, ap_details = self.get_accesspoint_details(bulk_config)
+        self.payload["access_point_details"] = ap_details
+        ap_update_list = []
+        common_config = {}
+        ap_output_list = []
+        if ap_exist and len(ap_details) > 0:
```

Comment:
If we add a new line before if, the code looks good.. Can we also add a log message?

    ap_exist, ap_details = self.get_accesspoint_details(bulk_config)
    self.payload["access_point_details"] = ap_details
    ap_update_list = []
    common_config = {}
    ap_output_list = []

    if ap_exist and len(ap_details) > 0:
        self.log("Access points exist. Total count: {}".format(len(ap_details)), "INFO")

        for each_ap in ap_details: